service/matching/poller/history.go

```diff
@@ -42,19 +42,19 @@ type (
 		RatePerSecond  float64
 		IsolationGroup string
 	}
-)
 
-type History struct {
-	// poller ID -> pollerInfo
-	// pollers map[pollerID]pollerInfo
-	history cache.Cache
+	History struct {
```

Comment:
nit: Less indentation is preferred so I don't favor grouping structs under a `type( ... )` section